### PR TITLE
Widget as WC plugin in ui-card

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1360
[Problem] HomeApp: change emebeded content to remote web clip
[Solution]
   - new widget - Card
   - examples

### CODE REVIEW

File: examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js [hunk 1]
```diff
@@ -1,11 +1,184 @@
-/* global tau */
+const tmpl = document.createElement("template");
+
+tmpl.innerHTML = `
```
Comment (singa2000) [inline examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js:4]: I just have a question.
There are the css and html code in the js file.
A general developer who distributes web clips, he have to include the css/html code into js fie??
Or it's just temporary?

Comment (lslachciak) [reply examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js:4]: how about separating them in future like in
https://stackoverflow.com/questions/55080103/how-to-separate-web-components-to-individual-files-and-load-them

Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js:4]: This commit is early draft. I try verify different ways to achieving the goal.

Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js:4]: Developer who distribute web clip can do as he needs. In my case I wanted to add template to js file. This plugin has been realized by Custom Element but other Web Clips as "TV Remote Control" has been made as html and css.

At the moment plugin need `html` file eg. `index.html` (example: `examples/mobile/HomeApp/webclip/apps-on-tv/index.html`) with
`<div class="ui-card"></div>` element.

File: examples/mobile/HomeApp/css/style.css [hunk 2]
```diff
@@ -130,26 +24,6 @@
...
-.app-breakfast-3 {
-    background-image: url(../images/800px-Korean.cuisine-Bibimbap-01.jpg);
-}
-
-.app-hotel-order {
-    display: flex;
-    align-self: center;
-}
-
 .app-dummy-payment input {
```
Comment (lslachciak) [inline examples/mobile/HomeApp/css/style.css:27]: this is part of widget too I think

File: examples/mobile/HomeApp/index.html [hunk 3]
```diff
@@ -34,256 +34,15 @@
...
-			</div>
-			<div class="ui-content-area app-weather-container">
-				<div class="app-progress-container">
-					<div class="ui-progress" data-type="indeterminatecircle" data-size="large"></div>
-				</div>
-				<a style="border-radius: 26px" class="weatherwidget-io" href="https://forecast7.com/en/37d57126d98/seoul/" data-label_1="Seoul" data-label_2="South Korea" data-theme="pure" >
-				</a>
-			</div>
+			<!-- Home App Plugins -->
+			<div class="ui-card" rel="card" data-src="webclip/weather/index.html"></div>
```
Comment (lslachciak) [inline examples/mobile/HomeApp/index.html]: rel attribute is precisely described by https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel
how about using another attribute?

Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/index.html]: I removed this attribute. Isn't required.

File: examples/mobile/HomeApp/css/style.css (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/css/style.css:27]: This is little complicated now. The Web Clip will open instant app as full page. This style is required by other page of Home App.

**Review by lslachciak — APPROVED** [review]

**Pull request merged**